R/datasheetSpatRaster.R

```diff
@@ -217,12 +217,12 @@ setMethod("datasheetSpatRaster", signature(ssimObject = "Scenario"),
   }
   
   # TO DO: make sure datasheet is spatial after opening
-  cMeta <- .datasheet(x, name = datasheet, optional = TRUE, 
+  cMeta <- datasheet(x, name = datasheet, optional = TRUE, 
```

Comment:
I think you want to leave the "." in front of `datasheet()`. This is an internal wrapper for the `datasheet()` function which guards against situations in which you have variable names with the same name as the function (in this case, one of the args is called `datasheet`). See internalWrappers.R for all the internal function names.

Is there a reason why you removed the the "."? Was it causing other issues?